MapboxNavigation/MapboxSpeechSynthesizer.swift

```diff
@@ -104,6 +110,39 @@ open class MapboxSpeechSynthesizer: NSObject, SpeechSynthesizing {
         audioPlayer?.stop()
     }
     
+    /**
+     Vocalize the provided audio data.
+     
+     This method is a final part of a vocalization pipeline. It passes audio data to the audio player. `instruction` is used mainly for logging and reference purposes. It's text contents do not affect the vocalization while the actual audio is passed via `instructionData`.
+     - parameter instruction: corresponding instruction to be vocalized. Used for logging and reference. Modifying it's `text` or `ssmlText` does not affect vocalization.
+     - parameter instructionData: audio data, as provided by `speechEngine`, to be played.
+     */
+    open func speak(instruction: SpokenInstruction, instructionData: Data) {
```

Comment:
For consistency with `speak(_:during:locale:)`, consider renaming this method to `speak(_:data:)`.